misc/scripts/update.sh

```diff
@@ -64,16 +67,17 @@ echo '
  / ____/ /_/ / /__(__  ) /_/ /_/ / / /
 /_/    \__,_/\___/____/\__/\__,_/_/_/
 '
-if [[ $USERNAME == "pacstall" ]] && [[ $BRANCH == "master" ]]; then
-	echo -e "[${BGreen}+${NC}] INFO: You are at version $(pacstall -V)"
-	echo -e "[${BYellow}*${NC}] WARNING: Be sure to check our GitHub release page to make sure you have no incompatible code: https://github.com/pacstall/pacstall/releases"
-else
-	echo -e "[${BYellow}*${NC}] WARNING: You are using a ${RED}development${NC} version of $(pacstall -V)"
```

Comment:
I would still tell the user is a dev version and ask for bug reports, but rephrasing it for the better, of course.